spec/unit/actions/rename_existing_client_projects_spec.rb

```diff
@@ -0,0 +1,52 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata/lcm/lcm2'
+require 'gooddata/lcm/actions/synchronize_etls_in_segment'
+
+describe GoodData::LCM2::RenameExistingClientProjects do
+  let(:gdc_gd_client) { double('gdc_gd_client') }
+  let(:domain) { double('domain') }
+  let(:client) { double('client') }
+  let(:project) { double('project') }
+
+  context 'client project is existing in input source' do
+    let(:params) do
+      params = {
+        gdc_gd_client: gdc_gd_client,
+        organization: domain,
+        clients: [
+          {
+            id: 'Id',
+            settings: [
+              {
+                name: 'lcm.title',
+                value: 'renamed project'
+              }
+            ]
+          }
+        ],
+        client_projects: {
+          'Id' => [client, project]
```

Comment:
The array should now become a hash like this: `{  segment_client: client, project: project }`